app/models/placement.rb

```diff
@@ -31,4 +31,6 @@ class Placement < ApplicationRecord
   accepts_nested_attributes_for :subjects, allow_destroy: true
 
   validates :school, :status, presence: true
+
+  scope :order_by_subject_school_name, -> { joins(:subjects, :school).order("subjects.name", "schools.name") }
```

Comment:
It looks like @Kizr used `.includes` rather than `.joins` in the support placements controller:

https://github.com/DFE-Digital/itt-mentor-services/blob/f584ba35165f75ec1b1bbb96ec934ec09a583305/app/controllers/placements/support/schools/placements_controller.rb#L6

I wonder if that'd work for our needs here?

```suggestion
  scope :order_by_subject_school_name, -> { includes(:subjects, :school).order("subjects.name", "schools.name") }
```